### DISCUSSION TITLE
The payment has passed. There is no subscription

### DISCUSSION THREAD

**Discussion by nekl3103** [community/community#45778] [Other Feature Feedback, Questions, & Ideas]
I paid for the subscription yesterday, but I had an error that no one contacted me in the next 24 hours, no one contacted me, the subscription was not activated why?

**Comment by jge162** [comment]
Which subscription are you referring to.

**Reply by nekl3103** [reply]
Subscribe to the Pro version on github

**Reply by jge162** [reply]
Here you go choose billing for help at [GitHub Support](https://support.github.com/request)
have a great rest of your day.

**Reply by jge162** [reply]
If you are satisfied with answer please consider marking answered. If not you can keep ticket open for more discussion. Thank you.

**Reply by nekl3103** [reply]
Thanks for the help

**Reply by jge162** [reply]
No problem hope it’s resolved for you do you can enjoy pro.
